yogstation/code/game/objects/items/holotool/modes.dm

```diff
@@ -0,0 +1,77 @@
+/datum/holotool_mode
+	var/name = "???"
+	var/sound
+	var/behavior
+	var/speed = 0.55
```

Comment:
make it 0.75, in line with https://github.com/yogstation13/Yogstation-TG/pull/3873